lib/recipe/presenter.rb

```diff
@@ -6,47 +6,38 @@ def initialize(recipe:)
       @recipe = recipe
     end
 
-    def present
-      output = <<~STR
+    def output
```

Comment:
This could take a `recipe` argument which might make it more obvious that the class is presenting a recipe, eg: `presenter.output(recipe: )`. It would make the data flow clearer.